### DISCUSSION TITLE
Is the same to declare plugins in Gemfile?

### DISCUSSION THREAD

**Discussion by MartinGonzalez** [fastlane/fastlane#17619] [Q&A] [answered]
I've seen that Pluginfile has the responsability to keep fastlane plugins dependencies.

Is it the same to not use a Pluginfile and put gems directly in Gemfile?
If not, what does Pluginfile?

Thanks!

**Comment by ainame** [comment, marked as answer]
> Is it the same to not use a Pluginfile and put gems directly in Gemfile?

Yes, it is. With Pluginfile fastlane offers some handy CLI tools to add/search for/install/update plugins!
https://docs.fastlane.tools/plugins/using-plugins/

**Reply by MartinGonzalez** [reply]
Thanks! :D
